primitives/arithmetic/src/fixed_point.rs

```diff
@@ -928,14 +931,14 @@ macro_rules! implement_fixed {
 			}
 		}
 
-		#[cfg(feature = "std")]
+		#[cfg(feature = "serde")]
 		impl sp_std::fmt::Display for $name {
 			fn fmt(&self, f: &mut sp_std::fmt::Formatter) -> sp_std::fmt::Result {
 				write!(f, "{}", self.0)
 			}
 		}
 
-		#[cfg(feature = "std")]
+		#[cfg(feature = "serde")]
```

Comment:
Necessary because Deserialize uses `from_string()`.

Reply:
Same here; we should just be able to remove the `#[cfg(feature)]`.